qfdmd/models.py

```diff
@@ -134,7 +134,7 @@ def url_carte_mauvais_etat(self):
 
     @cached_property
     def url_carte_bon_etat(self):
-        actions = "preter|emprunter|louer|mettreenlocation|donner|echanger|revendre"
+        actions = "preter|louer|mettreenlocation|donner|echanger|revendre"
```

Comment:
en bonne état tu ne veux pas la louer mais le mettre en location uniquement ?

```suggestion
        actions = "preter|mettreenlocation|donner|echanger|revendre"
```

Reply:
hum bonne question, je cette liste hardcodée et un peu foireuse, je vérifie mais en fait il faut toutes les actions "jai", donc il y a p-e mieux à faire